packages/lesswrong/lib/reviewUtils.tsx

```diff
@@ -67,3 +69,64 @@ export const currentUserCanVote = (currentUser: UsersCurrent|null) => {
   if (isEAForum && new Date(currentUser.createdAt) > new Date(annualReviewStart.get())) return false
   return true
 }
+
+const getPointsFromCost = (cost) => {
+  // the formula to quadratic cost from a number of points is (n^2 + n)/2
+  // this uses the inverse of that formula to take in a cost and output a number of points
+  return (-1 + Math.sqrt((8 * cost)+1)) / 2
+}
+
+const getLabelFromCost = (cost) => {
+  // rounds the points to 1 decimal for easier reading
+  return round(getPointsFromCost(cost), 1)
+}
+
+export const getCostData = ({costTotal=500}:{costTotal?:number}) => {
```

Comment:
Why does this take an object-argument? I mean, maybe fine for future-proofness, but this feels like it should just take a single argument that's a number.

Reply:
I thought I had to do that to give it a default value. Not sure if that was necessary.